docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -1288,6 +1294,8 @@ code: |
           }
       else:
         new_object.params = {}
+    else:
+      new_object.params = {}        
```

Comment:
Might be slightly better to just set `new_object.params = {}` before the `if person in people_quantities` on line 1285. Then you can get rid of this else and the else above, and it'll be certain to be set even if there are more ifs later.